src/Components/AddNote/AddNote.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+
+function AddNote({ onAddNote }) {
+  const [inputValue, setInputValue] = useState({
+    title: '',
+    content: '',
+  });
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const handelTitleChange = e => {
+    setInputValue(
+        {...inputValue,
+        title: e.target.value,}
+       );
+  };
```

Comment:
Same problem, let's use setState with callback in case we depend on the previous state.
```suggestion
  const handelTitleChange = e => {
    setInputValue(prevInput => ({...prevInput, title: e.target.value}));
  };
```


Read this article: https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state